### DISCUSSION TITLE
Which-key - shortcuts

### DISCUSSION THREAD

**Discussion by raylemon** [abougouffa/minemacs#109] [Q&A] [answered]
Is there a way to export all keyboard shortcuts from which-key? I will create a cheat sheet with whole commands. Thanks !

**Comment by abougouffa** [comment, marked as answer]
Désolé, je pensai que j'ai répondu à cette question.

Tu peux afficher la liste de tous les keybindings via `SPC h g` ou `M-x general-describe-keybindings`
